src/content/blog/2023-11-07-whats-coming-in-eslint-9.0.0.md

```diff
@@ -0,0 +1,124 @@
+
+* `checkstyle`
+* `compact`
+* `jslint-xml`
+* `junit`
+* `tap`
+* `unix`
+* `visualstudio`
+
+If you are using these formatters currently, you'll need to install the [standalone packages](https://github.com/fregante/eslint-formatters) for use with ESLint v9.0.0.
+
+### Removing `valid-jsdoc` and `require-jsdoc` rules
+
+Continuing with our theme of reducing ESLint's install size, we have decided to [remove the `valid-jsdoc` and `require-jsdoc` rules](https://github.com/eslint/eslint/issues/15820). These rules have been deprecated since ESLint v5.10.0 and removing them allows us to remove another dependency (`doctrine`). We recommend using the [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc) plugin instead.
+
+### Removing deprecated methods on `context` and `SourceCode`
+
+As we [announced in September](https://eslint.org/blog/2023/09/preparing-custom-rules-eslint-v9/), we will be removing a lot of deprecated methods from `context` and replacing them with methods on `SourceCode`. Please read the above mentioned post for more details on these changes.
+
+Additionally, we will be [removing the deprecated `SourceCode#getComments()` method](https://github.com/eslint/eslint/issues/14744).
+
+### Updating `eslint:recommended`
+
+The `eslint:recommended` configuration will be [updated](https://github.com/eslint/eslint/issues/17596) to include new rules that we feel are important, and to remove deprecated rules.
+
+### Changes to how you write rules
+
+To clean up and make rules easier to work with, we're making [two changes](https://github.com/eslint/eslint/issues/14709):
+
+1. Function-style rules will stop working in v9.0.0. Function-style rules are rules created by exporting a function from a file rather than exporting an object with a `create()` method.
+2. When a rule doesn't have `meta.schema` specified, a default schema of `[]` will be applied. This means that rules without a schema will be assumed to have no options, which in turn means that validation will fail if options are provided.
+
+You can read more about this change in the [RFC](https://github.com/eslint/rfcs/tree/main/designs/2021-schema-object-rules).
+
+### `reportUnusedDisableDirectives` enabled by default and more flexible
+
+The `reportUnusedDisableDirectives` option is undergoing several changes in v9.0.0:
+
+1. The `reportUnusedDisableDirectives` configuration option can now be configured as `"off"`, `"warn"`, and `"error"` in your `eslint.config.js` file. Going forward, `true` will be equivalent to `"warn"` and `false` will be equivalent to `"off"`.
+2. The `reportUnusedDisableDirectives` configuration option is enabled by default at the `"warn"` level.
+3. You can also change the severity level on the command line by using the new `--report-unused-disable-directives-severity` option.
+
+You can read more about this change in the [RFC](https://github.com/eslint/rfcs/tree/main/designs/2022-unused-disable-directive-flexible-config).
```

Comment:
This feature can and might be released in v8 (https://github.com/eslint/eslint/pull/17212). Only changing the default must be in v9.

Reply:
Actually, the default value can also be changed in v8, as it will only be changed in flat config mode.

Reply:
Ah ok. We do have the issue listed on the v9 board, so I'll remove it.